### PR TITLE
add changes to contractual form feature #38

### PR DESCRIPTION
**Pull request by fernandoamz**

### CODE REVIEW

File: app/controllers/contractuals_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  end
+
+  # POST /contractuals
+  # POST /contractuals.json
+  def create
+    @contractual = Contractual.new(contractual_params)
+
+    respond_to do |format|
+      if @contractual.save
+        format.html { redirect_to @contractual, notice: 'Contractual was successfully created.' }
```
Comment (diegosaul402) [inline app/controllers/contractuals_controller.rb]: Please localize this text.

File: app/controllers/contractuals_controller.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      end
+    end
+  end
+
+  # PATCH/PUT /contractuals/1
+  # PATCH/PUT /contractuals/1.json
+  def update
+    respond_to do |format|
+      if @contractual.update(contractual_params)
+        format.html { redirect_to @contractual, notice: 'Contractual was successfully updated.' }
```
Comment (diegosaul402) [inline app/controllers/contractuals_controller.rb]: Please localize this text.

File: app/controllers/contractuals_controller.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      end
+    end
+  end
+
+  # DELETE /contractuals/1
+  # DELETE /contractuals/1.json
+  def destroy
+    @contractual.destroy
+    respond_to do |format|
+      format.html { redirect_to contractuals_url, notice: 'Contractual was successfully destroyed.' }
```
Comment (diegosaul402) [inline app/controllers/contractuals_controller.rb]: Please localize this text.

File: app/views/contractuals/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: contractual, local: true) do |form| %>
+  <% if contractual.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(contractual.errors.count, "error") %> prohibited this contractual from being saved:</h2>
```
Comment (diegosaul402) [inline app/views/contractuals/_form.html.erb]: Please localize this text.

File: app/views/contractuals/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+  <tbody>
+    <% @contractuals.each do |contractual| %>
+      <tr>
+        <td><%= contractual.player_type %></td>
+        <td><%= contractual.agent %></td>
+        <td><%= contractual.nui %></td>
+        <td><%= link_to 'Show', contractual %></td>
+        <td><%= link_to 'Edit', edit_contractual_path(contractual) %></td>
+        <td><%= link_to 'Destroy', contractual, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```
Comment (diegosaul402) [inline app/views/contractuals/index.html.erb]: Please localize this text.

File: config/locales/en.yml [hunk 6]
```diff
@@ -45,6 +45,23 @@ en:
...
     team_update: 'Team was successfully updated.'
     teams: 'Teams'
+  
+  contractual: 
+    agent: 'Agent'
+    alert_message: 'Are you sure?'
+    button_send: 'Send'
+    contractual: 'Contractual'
+    destroy: 'Contractual was successfully destroyed.'
+    destroy: 'destroy'
```
Comment (8geonirt) [inline config/locales/en.yml]: This value is duplicated, removed the one you don't need

File: config/locales/es.yml [hunk 7]
```diff
@@ -46,6 +46,24 @@ es:
...
     teams: 'Equipos'
 
+  contractual: 
+    agent: 'Agente'
+    alert_message: '¿Estás seguro?'
+    back: 'Atrás'
+    button_destroy: 'Eliminar'
+    button_send: 'Enviar'
+    contractual: 'Contractual'
+    destroy: 'Contractual fue eliminado éxitosamente.'
```
Comment (8geonirt) [inline config/locales/es.yml]: Typo, it should be: `exitosamente`

File: config/locales/es.yml [hunk 8]
```diff
@@ -46,6 +46,24 @@ es:
...
+    edit: 'Editar'
+    edit_contractual: 'Editar contractual'
+    error_message: ' los siguientes campos no pueden estar vacios'
+    new_contractual: 'Nuevo contractual'
+    nui: 'NUI'
+    player_type: 'Tipo de jugador'
+    save: 'Contractual fue creado éxitosamente.'
+    show: 'Mostrar'
+    update: 'Contractual fue actualizado éxitosamente.'
```
Comment (8geonirt) [inline config/locales/es.yml]: No lleva acento

File: config/locales/es.yml [hunk 9]
```diff
@@ -46,6 +46,24 @@ es:
...
+    button_send: 'Enviar'
+    contractual: 'Contractual'
+    destroy: 'Contractual fue eliminado éxitosamente.'
+    edit: 'Editar'
+    edit_contractual: 'Editar contractual'
+    error_message: ' los siguientes campos no pueden estar vacios'
+    new_contractual: 'Nuevo contractual'
+    nui: 'NUI'
+    player_type: 'Tipo de jugador'
+    save: 'Contractual fue creado éxitosamente.'
```
Comment (8geonirt) [inline config/locales/es.yml]: Sin acento 😲

File: config/locales/es.yml [hunk 10]
```diff
@@ -59,3 +77,12 @@ es:
...
             end_date: 
               blank: 'La fecha de término no debe estar en blanco'
+        contractual:
+          attributes:
+            player_type:
+              blank: 'El tipo de jugador no debe estar en blanco'
+            agent:
+              blank: 'El agente no debe estar en blanco'
+            nui: 
+              blank: 'El nui no debe estar en blanco'
```
Comment (8geonirt) [inline config/locales/es.yml]: Si _nui_ es una sigla debe de ir en mayúsculas => NUI

**Pull request merged**